doc/api.html

```diff
@@ -1598,6 +1598,13 @@ <h3 id="massa.api.v1.LargestStakersContext">LargestStakersContext</h3>
                   <td><p>Slot </p></td>
                 </tr>
               
+                <tr>
+                  <td>in_downtime</td>
+                  <td><a href="#bool">bool</a></td>
+                  <td></td>
+                  <td><p>If the chain is in downtime, we will return the last snapshot before the downtime </p></td>
+                </tr>
+              
```

Comment:
Not sure why this is here, it was deprecated yesterday :). But I think it's only in the docs. @aoudiamoncef do you know what happened?

Reply:
gonna check if a rebase is needed

Reply:
this working branch is correctly based on main, so I dunno about the doc